profile/profile.yaml

```diff
@@ -0,0 +1,147 @@
+# Feel free to add more projects
+highlighted_projects:
+    #project name
+  - name: "Android Marketplace/ Auction app"
+    #URL is optional
+    git_url:
+    # The project description must start with an action verb (https://bit.ly/3guO98u)
+    description: "Collaborated with a team of four other members to build a mobile marketplace/ auction app where users
+    can buy, sell, track and own merchandise in a virtual space using Android studio and written in Java."
+  - name: "Course Registration Website"
+    git_url:
+    description: "Developed a simple frontend user interface for a course registration website alongside team members
+    using HTML, CSS and JavaScript while keeping Human Computer Interaction principles."
+  - name: "Course Registration Website"
+    git_url:
+    description: "Created a 3D hierarchical animation video game that modelled explorers in a gallery with
+    observable exhibits. It included 3D collision checks and texture mapping that brings the gallery to life.
+    Done in processing with the OpenGL API using Java. Included free-look feature which enables the explorer to look
+    around their environment when the mouse is dragged across the screen."
+
+#List of professional experience
+#The job description allows hiring managers to understand your job and the technologies you used(e.g. programming languages, libraries, algorithms, hardware, tools, etc.).
+#The description must be more than 25 words. If your experience is not software related, please focus on soft skills and achievements (e.g. team collaboration, problem-solving, improvements, communication).
+professional_experience:
+  #Write your Jarvis role as a professional experience rather than training.
+  - title: "Data Engineer"
+    company: "Jarvis Consulting Group"
+    duration: "2021-present"
+    description: "Developing multiple Software projects in a scrum environment following the Software Development Life
+    Cycle using the agile methodology. Projects are built with Linux, Core Java, SpringBoot, Python, and SQL."
+  - title: "Technical Customer Support Advisor"
+    company: "24-7 Intouch"
+    duration: "2020-2021"
+    description: "Mentored new advisors to equip them with the skills and expertise needed to fulfill their work
+    requirements. Resolved technical issues from customers and provided solutions by troubleshooting, diagnosing,
+    setting up repairs and more."
+  - title: "Student Tutor"
+    company: "Nimbus Tutoring"
+    duration: "2018-2020"
+    description: "Conducted one-on-one tutoring sessions for fellow students in Statistics and Calculus to provide
+    solutions in areas of difficulty and consequently, improve their grades."
+  - title: "Software Developer Intern"
+    company: "ActivEdge Technologies"
+    duration: "2017"
+    description: "Strengthened and mastered programming skills by joining the Technology team to develop programs in C,
+    Java, JavaScript, HTML and database concepts"
```

Comment:
too short. did you develop a project/product? agile? libraries/frameowrks? soft skills?